dali/operators/python_function/dltensor_function.cc

```diff
@@ -32,22 +32,23 @@ DALI_SCHEMA(DLTensorPythonFunctionImpl)
     .MakeInternal();
 
 DALI_SCHEMA(DLTensorPythonFunction)
-    .DocStr(R"code(Execute a python function that operates on DLPack tensors.
+    .DocStr(R"code(Executes a Python function that operates on DLPack tensors.
+
 The function should not modify input tensors.
 
-In case of the GPU operator it is a user's responsibility to synchronize the device code with DALI.
-This can be accomplished by synchronizing DALI's work before the operator call
-with the `synchronize_stream` flag (true by default) and then making sure
-the scheduled device tasks are finished within the operator call.
-Alternatively, the gpu code can be done on the DALI's stream
-which may be determined by calling the `current_dali_stream()` function.
-In this case, the `synchronize_stream` flag can be set to false.)code")
+For the GPU operator, it is your responsibility to synchronize the device code with DALI.
+To synchronize the device code with DALI, synchronize DALI’s work before the operator call
+with the ``synchronize_stream`` flag, set to tTrue by default and ensure that the scheduled device
```

Comment:
```suggestion
with the ``synchronize_stream`` flag (enabled by default) and ensure that the scheduled device
```
I think it's easier to follow this way.

Reply:
Done